README.md

```diff
@@ -491,7 +491,7 @@ toContainHTML(htmlText: string)
 ```
 
 Assert whether a string representing a HTML element is contained in another
-element:
+element. The string should contain valid html, and not any substrings of html.
```

Comment:
"substrings of html" is a bit unclear to me. It sounds like "<p></p>" would be considered a "substring" but that's valid HTML. An example would help here.

Reply:
I agree this is probably not the clearest terms. But the examples are below. Do you think we need examples of what won't work? Like for instance, this:

```js
// These won't work
render(<div class="test">text</div>)
expect(element).toContainHTML('</div>')
expect(element).toContainHTML('class="test"')
```

Reply:
I think "partial HTML" or "incomplete HTML" is more fitting here. 

Oh and move the comment since the render does work.
```diff
-// These won't work
render(<div class="test">text</div>)
+// These won't work
expect(element).toContainHTML('</div>')
expect(element).toContainHTML('class="test"')
```